bibigrid-core/src/main/java/de/unibi/cebitec/bibigrid/core/util/ShellScriptCreator.java

```diff
@@ -43,10 +46,6 @@ public static String getUserData(Configuration config, boolean base64) {
         userData.append("log \"set hostname\"\n");
         userData.append("hostname -b $(iplookup $localip)\n");
         userData.append("fi\n");
-        // disableAptDailyService
-        userData.append("log \"disable apt-daily.(service|timer)\"\n");
```

Comment:
Why is the log removed?

Reply:
When disable `apt daily` as early as possible there is no log functionality available yet.